modules/ding_seo/ding_seo.features.user_permission.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * ding_seo.features.user_permission.inc
+ */
+
+/**
+ * Implements hook_user_default_permissions().
+ */
+function ding_seo_user_default_permissions() {
```

Comment:
This will have to be setup using the module secure_permissions.

1. You could add it to the `ding_permissions_secure_permissions()` in ding_permissions module.
2. You could implement hook in ding_seo  `ding_seo_secure_permissions()`. Like ding_section in `ding_sections_secure_permissions()`

I would prefer option 1. and depend on ding_permission module.